governance/parameter-docs/param-dsr.md

```diff
@@ -0,0 +1,52 @@
+# $HumanReadableName
+
+```
+Alias: NA
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs:https://docs.makerdao.com/smart-contract-modules/rates-module/pot-detailed-documentation
+```
+
+## Description
+* What is this parameter?
+The Dai Savings Rate (DSR) is a variable rate of accrual earned by locking Dai in the DSR smart contract. Dai holders can earn savings automatically and natively while retaining control of their Dai. The DSR smart contract has no withdrawal limits, deposit limits, or liquidity constraints.
+
+Interest will be distributed to users every second at a growth rate of x% per second, but in the UI rates are displayed annually. For example, if the DSR was 2%, assuming the user put in 100 Dai, at the end of the first year they would have 102.00, and at the end of the second year they would have 104.04.
```

Comment:
"Interest will be distributed to users every second at a growth rate of x% per second" - This might be misleading, the interest isn't actually distributed until a user withdraws. Interest accrues to users each block, but rates are displayed and set annually. 

"but in the UI rates are displayed annually." - Governance sets the rates in annual terms too (though smart contract value isn't.)